assets/javascripts/discourse/components/group-assignments-menu-item.js.es6

```diff
@@ -3,6 +3,11 @@ export default Ember.Component.extend({
 
   init() {
     this._super(...arguments);
-    this.set("canAssign", this.currentUser && this.currentUser.can_assign);
+    this.set(
+      "canAssign",
```

Comment:
We should rename this to "displayAssignTab" or something. Because it doesn't mean 'canAssign' any more

Reply:
Sure, ill rename this